examples/js/animation/MMDAnimationHelper.js

```diff
@@ -945,6 +945,8 @@ THREE.MMDAnimationHelper = ( function () {
 			}
 
 			if ( this.currentTime < this.delayTime ) return false;
+			
+			if ( ( this.currentTime - this.delayTime ) > this.audioDuration ) return false;
```

Comment:
Can "`( this.currentTime - this.delayTime ) > this.audioDuration`" be `true` here?

Reply:
Yes, when `params.sync` is `true` (and the default value is true), `AudioManager.duration` may be reset to a larger value than `audioDuration + delayTime`.

In the `_syncDuration` method (line number 747)：


			if ( audioManager !== null ) {

				audioManager.duration = max;

			}

Reply:
Oh, right. Could you add a comment "duration can be bigger than audioDuration + delayTime because of sync configuration" or something?